CVLByExample/Teams/NoPreserved.spec

```
@@ -5,9 +5,7 @@ methods {
     function leaderOf(uint8) external returns (address) envfree;
 }
 
-
 /** @title If a team does not exist it has not players
  *  This invariant cannot be proven without a preserved block.
  */
-invariant nonExistTeamHasNoPlayers(uint8 teamId, address player)
-    (teamId != 0 && leaderOf(teamId) == 0) => teamOf(player) != teamId;
+invariant nonExistTeamHasNoPlayers(uint8 teamId, address player) (teamId != 0 && leaderOf(teamId) == 0) => teamOf(player) != teamId ;
```

MUST: Can we _not_ add a space at the end of the expression?
```suggestion
invariant nonExistTeamHasNoPlayers(uint8 teamId, address player) (teamId != 0 && leaderOf(teamId) == 0) => teamOf(player) != teamId;
```